instrumentation/opentelemetry-instrumentation-botocore/tests/test_botocore_instrumentation.py

```diff
@@ -144,29 +139,42 @@ def test_s3_put(self):
         location = {"LocationConstraint": "us-west-2"}
         s3.create_bucket(Bucket="mybucket", CreateBucketConfiguration=location)
         s3.put_object(**params)
+        s3.get_object(Bucket="mybucket", Key="foo")
 
         spans = self.memory_exporter.get_finished_spans()
         assert spans
-        span = spans[0]
-        self.assertEqual(len(spans), 2)
-        self.assertEqual(span.attributes["aws.operation"], "CreateBucket")
-        assert_span_http_status_code(span, 200)
+        self.assertEqual(len(spans), 3)
+        create_bucket_attributes = dict(spans[0].attributes)
         self.assertEqual(
-            span.resource,
-            Resource(
-                attributes={"endpoint": "s3", "operation": "createbucket"}
-            ),
+            create_bucket_attributes,
+            {
+                "aws.operation": "CreateBucket",
+                "aws.region": "us-west-2",
+                "aws.service": "s3",
+                "http.status_code": 200,
+            },
         )
-        self.assertEqual(spans[1].attributes["aws.operation"], "PutObject")
+        put_object_attributes = dict(spans[1].attributes)
         self.assertEqual(
-            spans[1].resource,
-            Resource(attributes={"endpoint": "s3", "operation": "putobject"}),
+            put_object_attributes,
+            {
+                "aws.operation": "PutObject",
+                "aws.region": "us-west-2",
+                "aws.service": "s3",
+                "http.status_code": 200,
+            },
         )
-        self.assertEqual(spans[1].attributes["params.Key"], str(params["Key"]))
+        self.assertTrue("params.Body" not in spans[1].attributes.keys())
+        get_object_attributes = dict(spans[2].attributes)
         self.assertEqual(
```

Comment:
I'm not sure why `RequestId` doesn't return here, but I raise an issue to ask if my setup is wrong.

https://github.com/spulec/moto/issues/3442

Reply:
Got a reply confirming that the `RequestId` is missing because its simply not added to the S3 "mock" of the service in `moto`, so this is the best we can do right now :)